hyp3_gamma/__main__.py

```diff
@@ -1,55 +1,68 @@
 """
-rtc_gamma processing for HyP3
+HyP3 plugin for processing SAR data with GAMMA
 """
 import glob
 import logging
 import os
-import re
 import sys
 from argparse import ArgumentDefaultsHelpFormatter, ArgumentParser
 from pathlib import Path
 from shutil import make_archive
+from zipfile import ZipFile
 
-from hyp3lib import GranuleError
 from hyp3lib.aws import upload_file_to_s3
 from hyp3lib.fetch import download_file, write_credentials_to_netrc_file
 from hyp3lib.image import create_thumbnail
 from hyp3lib.scene import get_download_url
 from hyp3lib.util import string_is_true
-from hyp3proclib import (
-    extra_arg_is,
-    failure,
-    get_extra_arg,
-    success,
-    upload_product,
-)
-from hyp3proclib.db import get_db_connection
-from hyp3proclib.file_system import cleanup_workdir
-from hyp3proclib.logger import log
-from hyp3proclib.proc_base import Processor
 from pkg_resources import load_entry_point
 
-import hyp3_rtc_gamma
-from hyp3_rtc_gamma.rtc_sentinel import rtc_sentinel_gamma
+from hyp3_gamma import __version__
+from hyp3_gamma.insar.ifm_sentinel import gamma_process
+from hyp3_gamma.rtc.rtc_sentinel import rtc_sentinel_gamma
+
+log = logging.getLogger(__name__)
 
 
 def entry():
     parser = ArgumentParser(prefix_chars='+', formatter_class=ArgumentDefaultsHelpFormatter)
+    parser.add_argument('++version', action='version', version=f'hyp3_gamma {__version__}')
     parser.add_argument(
-        '++entrypoint', choices=['hyp3_rtc_gamma', 'hyp3_rtc_gamma_v2'], default='hyp3_rtc_gamma',
-        help='Select the HyP3 entrypoint version to use'
+        'process', choices=['rtc', 'insar'],
+        help='Select the HyP3 process to use'
     )
     args, unknowns = parser.parse_known_args()
 
-    sys.argv = [args.entrypoint, *unknowns]
+    sys.argv = [args.process, *unknowns]
     sys.exit(
-        load_entry_point('hyp3_rtc_gamma', 'console_scripts', args.entrypoint)()
+        load_entry_point('hyp3_gamma', 'console_scripts', args.process)()
     )
 
 
-# v2 functions
-def main_v2():
-    parser = ArgumentParser()
+def find_png(product_dir):
```

Comment:
This function should be retired, it was previously used to find the single `png` that would be used as the browse image for hyp3 v1